import java.lang.reflect.InvocationTargetException;

public class Test {

 	public static void main(String[] args) {
 		
		try {
			Helper.test("moveXPlacesRemoveYNodes");
		} catch (Exception e) {
			e.printStackTrace();
		}
	}
}
